includes/Modules/Analytics/proxy-service-analytics-provisioning.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Class Google\Site_Kit\Modules\Analytics\Google_Service_AnalyticsProvisioning
+ *
+ * @package   Google\Site_Kit\Modules\Analytics
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Modules\Analytics;
+
+use Google\Site_Kit_Dependencies\Google_Service_Analytics;
+use Google\Site_Kit_Dependencies\Google_Client;
+
+/**
+ * Class for Analytics Provisioning service.
+ *
+ * @since n.e.x.t
+ * @access private
+ * @ignore
+ */
+class Google_Service_AnalyticsProvisioning extends Google_Service_Analytics {
+	/**
+	 * Constructs the internal representation of the Analytics service.
+	 *
```

Comment:
Missing since tag here.

Reply:
Added in https://github.com/google/site-kit-wp/pull/1512/commits/6a9a99a266df117cbc56e26e125b069b0356f6d7.